guides/_subsections/view-targets-with-topology.md

```diff
@@ -0,0 +1,112 @@
+## [Use Topology View](#use-topology-view)
+
+**Topology View** provides a visual presentation of the all discovered JVM applications and all resources that are associated with them. It also allows users to perform actions on one or multiple targets.
+
+{% include_relative _subsections/common/navigate-to-topology.md %}
+
+### [View all target JVMs](#view-all-target-jvms)
+
+<ol>
+    {% capture topology-graphview-guide-text %}
+    By default, an interactive <b>Graph View</b> of discovered target JVMs (<i>nodes</i>) are shown nested their associated groups (<i>surrounding lines</i>), for example, Pod, or Realm (i.e. discovery platform). 
```

Comment:
nested **within** ?

Reply:
and I think Pod and Realm should be special some sort of way (italicized?)

Reply:
Actually, I think we need a little more explanation on what a Realm is here. Users will probably understand well enough what the intermediate node types in between the Node and the target JVM, ex. if they're deploying in OpenShift they're probably already at least somewhat familiar with the Pod, Deployment, Namespace that they'll see appear, but Realm will be unfamiliar. "i.e. discovery platform" is in the right line for a hint but I don't know if a typical user is going to know what we mean by this. Also, if they're using the Cryostat Agent, each of those is its own Realm too, but it doesn't sound like it's a "platform".

I guess we just need a short explainer that Realm == "discovery plugin instance", including the built-in internal plugins like Kube API and Custom Targets.

Reply:
How about now?

Reply:
Sounds good